### PR TITLE
Update docs and add state for junos_lldp_global

### PR DESCRIPTION
**Pull request by danielmellado**
This commit adds gathered state for junos_lldp_global and updates the
documentation to the new resource modules versioning for collections.

### CODE REVIEW

**Comment by pabelanger** [comment]
recheck

File: plugins/modules/junos_lldp_global.py [hunk 1]
```diff
@@ -30,15 +30,12 @@
...
-
-DOCUMENTATION = """module: junos_lldp_global
-short_description: Manage link layer discovery protocol (LLDP) attributes on Juniper
-  JUNOS devices.
+ANSIBLE_METADATA = {"metadata_version": "1.1", "supported_by": "network"}
+
+DOCUMENTATION = """
+---
+module: junos_lldp_global
+short_description: Link layer discovery protocol (LLDP) JUNOS resource module.
```
Comment (samccann) [inline plugins/modules/junos_lldp_global.py:38]:
```suggestion
short_description: Link layer discovery protocol (LLDP) JUNOS resource module
```

**Comment by danielmellado** [comment]
recheck

**Comment by danielmellado** [comment]
recheck

**Pull request merged**